src/lib/config.js

```diff
@@ -20,10 +20,7 @@ module.exports = {
     SESSION_URL: process.env.REDIS_SESSION_URL,
     PORT: process.env.REDIS_PORT || 6379,
   },
-  ORDNANCE: {
-    ORDNANCE_SURVEY_SECRET: process.env.ORDNANCE_SURVEY_SECRET,
-    ORDNANCE_API_URL:
-      process.env.ORDNANCE_SURVEY_URL ||
-      "https://api.os.uk/search/places/v1/postcode?",
+  POSTCODE_API: {
+    POSTCODE_URL: process.env.POSTCODELOOKUP_URL || "http://localhost:5009",
```

Comment:
this can be refactored more as the postcode base url is the same as the base api base url.